frontend/catalyst/passes/builtin_passes.py

```diff
@@ -1018,8 +1025,7 @@ def circuit():
 
 
 def ppm_specs(fn):
```

Comment:
This is completely supplanted by the new `qml.spec` feature.  Might want to direct users to use `qml.spec` here.